app/Models/Track.php

```diff
@@ -637,7 +648,7 @@ public function isPublished()
 
     protected function getMasterTrackFile() : TrackFile
     {
-        return $this->trackFiles->where('is_master', true)->first();
+        return $this->trackFiles()->where('is_master', true)->first();
```

Comment:
I presume this was made for the same reason as [this change](https://github.com/Poniverse/Pony.fm/pull/105/files#r76924440).